src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -26,6 +26,7 @@ illegal.type=Declaring variables, return values or parameters of type ''{0}'' is
 inline.conditional.avoid=Avoid inline conditionals.
 instantiation.avoid=Instantiation of {0} should be avoided.
 magic.number=''{0}'' is a magic number.
+mismatch.package.directory=Package name is not same as directory.
```

Comment:
no change to message is required for now.
As diff report is provided we will see how easy to understand violation.